<launch>
  <node name="cv_camera" pkg="cv_camera" type="cv_camera_node" output="screen">
    <param name="rate" value="60" />
    <param name="device_id" value="1" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/cv_camera/image_raw"/>
    <param name="autosize" value="true" />
  </node>
</launch>
